src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Interop/JavaScriptExports.cs

```diff
@@ -24,15 +24,15 @@ public static void CallEntrypoint(JSMarshalerArgument* arguments_buffer)
                 arg_1.ToManaged(out IntPtr entrypointPtr);
                 if (entrypointPtr == IntPtr.Zero)
                 {
-                    throw new MissingMethodException("Missing entrypoint");
+                    throw new MissingMethodException(SR.Format(SR.MissingEntrypoint));
```

Comment:
extra SR.Format

Reply:
the other PR will be merged first and will replace this